docker/run-tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash -xe
+
+flake8 bedrock lib tests
```

Comment:
could do gulp lint here too. Have to gulp test in circle.yml because the docker container doesn't have the necessary browsers installed for those tests (and I couldn't get it working). So if I have to have the gulp stuff in circle anyway might as well do it there. I did have the flake8 thing in a separate `make lint` command, but this seemed more efficient, and it's not a bad idea to have people who run `make test` locally run it through flake8.